bundles/org.openhab.binding.intellicenter2/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,150 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="intellicenter2"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<bridge-type id="bridge">
+		<label>IntelliCenter 2 Bridge</label>
+		<description>Pentair IntelliCenter 2 Bridge</description>
+		<properties>
+			<property name="vendor">Pentair</property>
+		</properties>
+
+		<config-description>
+			<parameter name="hostname" type="text" required="true">
+				<context>network-address</context>
+				<label>Hostname</label>
+				<description>Hostname or IP address of the device</description>
+			</parameter>
+			<parameter name="port" type="integer" required="true">
+				<context>host</context>
+				<label>Port</label>
+				<description>Port of the device, default to 6681</description>
+			</parameter>
+		</config-description>
+	</bridge-type>
+
+	<thing-type id="pool">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="bridge"/>
+		</supported-bridge-type-refs>
+
+		<label>IntelliCenter2 Pool</label>
+		<description>Pool for IntelliCenter2 Binding</description>
+
+		<channels>
+			<channel id="current-temperature" typeId="temperature">
```

Comment:
You could consider supplying separate channel type definitions for current temperature and target temperature. This would make it possible to supply default semantic tags (Temperature + Measurement/Setpoint).

Reply:
Done.